doc/modules/calibration.rst

```diff
@@ -240,6 +240,26 @@ probabilities, the calibrated probabilities for each class
 are predicted separately. As those probabilities do not necessarily sum to
 one, a postprocessing is performed to normalize them.
 
+The parameters ``class_weight`` and ``sample_weight`` can be used to
+respectively set relative importance to classes and to individual samples.
+
+:class:`CalibratedClassifierCV` can handle such unbalanced dataset with the
+``class_weight`` parameter. ``class_weight`` has to be provided as a
+dictionary of the form ``{class_label : value}``, where value is a strictly
+positive floating point, or as ``class_weight='balanced'`` which will
+automatically adjust weights inversely proportional to class frequencies in
+the input data.
+
+**Setting ``class_weight`` parameter in :class:`CalibratedClassifierCV`,
+will only affect the training of the chosen regressor (`'isotonic'` or `'sigmoid'`).**
```

Comment:
The bold policy misses the link to the function in the [rendering](https://170189-843222-gh.circle-artifacts.com/0/doc/modules/calibration.html#multiclass-support). Perhaps a [note](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=note#directive-note) directive would be suitable here.